JM/CF/Scripts/3_Game/CommunityFramework/Timer/CF_TimerBase.c

```diff
@@ -0,0 +1,202 @@
+class CF_TimerBase : Managed
+{
+	private static ref CF_TimerBase s_Root;
+	private ref CF_TimerBase m_Next;
+	private CF_TimerBase m_Prev;
+	private bool m_IsActive;
+	private bool m_Destroy;
+
+	protected float m_Interval; // Seconds
+	protected float m_DeltaTime; // Seconds
+	protected int m_Time; // Milliseconds
```

Comment:
Storing `m_Time` in seconds will lead to floating-point errors when the timer/server has been alive for more than 2.5 hours. I would rather store it all in seconds but I'm keeping that in mind.

Reply:
Ok, storing time as ms in an int variable is enough for 25 days of runtime. Since we have the floating-point issue with all of the member vars, I would suggest we change all 3 to int so people do not need to look up what is *1000 or /1000 ... It is very common for ms intervals to be passed as ints and not floats in other languages, so I think this would be no issue.

Reply:
I can make it so member variaes are ints and milliseconds. 

But the arguments passed into the OnUpdate and OnTick methods will continue to remain floats and be in seconds, to keep consistent with every other Update method within vanilla  DayZ scripts

Reply:
The problem is that it is super ugly to have float parameters to pass in but in the function logic, you suddenly need to divide or multiply by 1000 because the member vars you interact with stored them this way. So no, I think it should be all ints.

Reply:
No valid use-case or equation works with milliseconds. They all work seconds.

Will just rename the `SetTimeElapsed` to `SetTimeElapsedMS` and `GetTimeElapsed` to `GetTimeElapsedMS`, keeping everything else as floating point numbers.

Reply:
What about we internally use an int to store it as m_ElapsedMS and also provide m_Elasped as float in seconds, that way people can choose what floats the boat. And the parameters are all gonna be in float.
We just update the float with `m_Elasped = m_ElapsedMS / 1000.0;`